pkg/storage/system_intake.go

```diff
@@ -67,9 +70,12 @@ func (s *Store) CreateSystemIntake(intake *models.SystemIntake) (*models.SystemI
 			:process_status,
 			:ea_support_request,
 			:existing_contract,
+		    :grt_review_email_body,
+			:requester_email_address,
 		    :created_at,
 		    :updated_at,
-			:submitted_at
+			:submitted_at,
+			:reviewed_at
```

Comment:
Please change `reviewed_at` to `decided_at` to better align with what we've sent to CEDAR. It's not imperative, but for intakes, they only get a decision, not a review (for retry).